bugbug/repository.py

```diff
@@ -128,6 +133,19 @@ def download_commits(repo_dir):
 
         author_experience[commit] = commits_by_author[commit.author]
 
+    for i in range(commits_num):
+        commits_by_author_3_months[commits[i].author] = 0
+        for j in range(i + 1, commits_num):
+            if commits[i].author == commits[j].author:
+                res = commits[i].date - commits[j].date
+                if hasattr(res, 'months') and res.months <= 3:
```

Comment:
Just use 90 days, easier to handle

Reply:
The difference between two datetime object have either > 90 days or > 3 months. I've added both cases.

Reply:
Mmmmh, the timedelta object should have a `days` property, not a `months`.